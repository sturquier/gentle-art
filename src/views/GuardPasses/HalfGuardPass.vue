<template>
  <h1>Half guard pass</h1>
</template>
