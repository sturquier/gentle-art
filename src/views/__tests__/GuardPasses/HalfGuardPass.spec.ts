import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import HalfGuardPass from '../../GuardPasses/HalfGuardPass.vue'

describe('HalfGuardPass', () => {
  it('renders properly', () => {
    const wrapper = mount(HalfGuardPass)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
