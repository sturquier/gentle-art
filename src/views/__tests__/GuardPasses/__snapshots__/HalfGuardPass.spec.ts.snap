// Vitest Snapshot v1

exports[`HalfGuardPass > renders properly 1`] = `"<h1>Half guard pass</h1>"`;
